test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -276,12 +280,11 @@ Describe "Import-Module from CompatiblePSEditions-checked paths" -Tag "CI" {
             & "Test-$ModuleName" | Should -Be $Result
         }
 
-        It "Fails to import incompatible modules from the module path with PSEdition <Editions>" -TestCases $failCases -Skip:(-not $IsWindows) {
+        It "Successfully imports incompatible modules from the module path with PSEdition <Editions> using WinCompat" -TestCases $failCases -Skip:(-not $IsWindows) {
             param($Editions, $ModuleName, $Result)
 
-            {
-                Import-Module $ModuleName -Force -ErrorAction 'Stop'; & "Test-$ModuleName"
-            } | Should -Throw -ErrorId "Modules_PSEditionNotSupported,Microsoft.PowerShell.Commands.ImportModuleCommand"
+            Import-Module $ModuleName -Force -ErrorAction 'Stop'
+            & "Test-$ModuleName" | Should -Be $Result
```

Comment:
Do we also wants tests on the `PrivateData.ImplicitRemoting` field, since that's a contract we depend on?

Reply:
I think it is already covered cause most implicit remoting tests will break if something happens to this field.